examples/experimental/dagster-airlift/dagster_airlift/core/sensor/sensor_builder.py

```diff
@@ -138,23 +145,24 @@ def airflow_dag_sensor(context: SensorEvaluationContext) -> SensorResult:
 
 
 def sorted_asset_events(
-    all_materializations: List[Tuple[float, AssetMaterialization]],
+    events: Sequence[AssetMaterialization],
     repository_def: RepositoryDefinition,
 ) -> List[AssetMaterialization]:
     """Sort materializations by end date and toposort order."""
     topo_aks = repository_def.asset_graph.toposorted_asset_keys
+    timestamps_and_events = [(get_timestamp_from_asset_event(event), event) for event in events]
     return [
         sorted_mat[1]
         for sorted_mat in sorted(
-            all_materializations, key=lambda x: (x[0], topo_aks.index(x[1].asset_key))
+            timestamps_and_events, key=lambda x: (x[0], topo_aks.index(x[1].asset_key))
```

Comment:
a comment that indicates what this sorted order ends up being would be good. lambda is hard to parse